website/project/views/comment.py

```diff
@@ -17,30 +18,69 @@
 
 
 @file_updated.connect
-def update_comment_root_target_file(self, node, event_type, payload, user=None):
-    if event_type == 'addon_file_moved':
+def update_file_guid_referent(self, node, event_type, payload, user=None):
```

Comment:
Mostly just everything inside the `if` statements would be broken into functions. Maybe just inside the terminal `if` statements.